schema/deploy/function_application_revision_application_revision_status.sql

```diff
@@ -0,0 +1,24 @@
+-- Deploy ggircs-portal:function_application_revision_application_revision_status to pg
+-- requires: table_application_revision_status
+
+begin;
+
+create or replace function ggircs_portal.application_revision_application_revision_status(application_revision ggircs_portal.application_revision)
+    returns ggircs_portal.application_revision_status
+as
+$function$
+declare
+begin
+    return (
+        select row(_application_revision_status.*)
+        from ggircs_portal.application_revision_status as _application_revision_status
+        where _application_revision_status.application_id = application_revision.application_id
+        and _application_revision_status.version_number = (select max(version_number) from ggircs_portal.application_revision as _application_revision
```

Comment:
shouldn't this be `and _application_revision_status.version_number = application_revision.version_number`

Reply:
yup